mypy.ini

```diff
@@ -0,0 +1,8 @@
+[mypy]
+ignore_missing_imports = True
+disallow_incomplete_defs = True
+disallow_untyped_defs = True
+disallow_untyped_calls = True
+
+[mypy-tests.miscutils.*]
```

Comment:
Do you know a good way to have mypy ignore test files other than adding them to a package?

Reply:
I've tried something like `mypy-test_*` to no avail.

Reply:
It looks like mypy requires files to be in a module in order to ignore them.